topics/data-management/tutorials/0013_file-naming/slides.html

```diff
@@ -0,0 +1,187 @@
+---
+layout: rdmbites_slides
+logo: /assets/images/elixir-uk.png
+video: true
+
+title: "How to Name Files"
+questions:
+objectives:
+key_points:
+contributors:
+  - robertmand
+  - bfranicevic
+  - saramorsy
+  - kpoterlowicz
+
+voice:
+  id: Amy
+  lang: en-GB
+  neural: true
+---
+
+## Learning Objective and Notes
+
+- To be able to create filenames that are machine-readable
+
+- To understand how filenames can be used to group files
+
+- To be able to create filenames that are human-readable
+
+- Although we refer to file naming, information in this RDMbite refers also to directory/folder naming
+
+???
+
+- In this RDM bite you will learn to how name files so that they can be read and interpreted by yourself and others.  
+
+- Filenames that are machine readable can be easily read and interpreted by code. 
+
+- Human readable filenames help you and others record and understand the contents of a file.  
+
+- You will also learn that filenames can contain punctuation enabling groups of files to be ordered and searched.
+
+---
+
+## Characters in Filenames to Avoid: Special Characters
+
+.left[Special characters to avoid]
+
+![special characters to avoid](../../images/0013/01.png)
+
+.left[Three special characters that can be used to delimit words in your filename]
+
+![special characters to use](../../images/0013/02.png)
+
+???
+
+- A filename can be both human and machine readable and as a general rule the use of special characters is avoided.  
+
+- So, what is a special character?  
+
+- A special character is any character that isn’t a number or a letter.  
+
+- Although we avoid the use of special characters in filenames, we will explain later that the use of the full stop, the underscore, and the minus sign are commonplace and useful.
+
+---
+
+## Characters in Filenames to Avoid: Special Characters
+
+.left[The forward slash (/) denotes a directory]
+
+![cat command](../../images/0013/03.png)
+
+???
+
+- Special characters are often used by programming languages to denote a special property or function.
+
+- One example you may know is the forward slash which denotes a directory.  A filename containing a forward slash character causes problems.  
+
+- In this example the Unix cat command is used to read the contents of a file.   Here the command throws an error because the filename contains a forward slash.  The command instead tries to look for a file called post qc dot txt under a directory called pre. 
+
+## Characters to Avoid: Spaces
+
+.left[Space characters are interpreted as spaces between different files]
+
+![cat command](../../images/0013/04.png)
```

Comment:
I know most people will access the videos you're putting out, but if you want to link to the slides for accessibility reasons then maybe this is a concern: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.